README.rst

```diff
@@ -490,6 +490,35 @@ You can though ask vim to run ``darker`` on file save with the following in your
 
 Vim should automatically reload the file.
 
+Emacs
+-----
+
+You can integrate with Emacs using Steve Purcell's `emacs-reformatter`__ library.
+
+Using `use-package`__ :
```

Comment:
In the documentation, I've used the [modern English-language printing](https://en.wikipedia.org/wiki/Colon_(punctuation)#Spacing) rules, so no space before a colon:
```suggestion
Using `use-package`__:
```